Resources/Textures/Objects/Fun/spraycans.rsi/meta.json

```diff
@@ -70,6 +79,22 @@
     {
       "name": "equipped-BELT",
       "directions": 4
+    },
+    {
+      "name": "clown-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "clown-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-left",
+      "directions": 4
     }
   ]
 }
```

Comment:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/5d2e1702-d3a1-4d3b-845c-3d26b4517e08)

I checked, there is currently no way to paint toolboxes and other things that items are automatically put into when clicked. I think we need to add Verbs for paint cans to get a paint workaround for such cases

Reply:
yea will probably need a verb to paint containers.

Reply:
Now has a verb which lets you right click and right click paint remove. Ideal for painting entites that are containers.